app/models/order.rb

```diff
@@ -5,6 +5,8 @@ class Order < ApplicationRecord
   has_many :items, through: :item_orders
   belongs_to :user
 
+  enum status: %w(pending packaged shipped cancelled)
```

Comment:
This is so great and would've been so useful for me and the stories I'm working on.